docs/plugins/mongodb.md

```diff
@@ -0,0 +1,40 @@
+# `mongodb` plugin
+
+The `mongodb` plugin consumes [MongoDB](https://www.mongodb.com/) log entries from the local filesystem and outputs parsed entries.
+
+## Configuration Fields
+
+| Field | Default | Description |
+| --- | --- | --- |
+| `source` | `file` | Use this field to specify where your logs are coming from. When choosing the 'file' option, the agent reads in logs from the log paths specified below.  When choosing the 'Kubernetes' options, the agent reads logs from /var/log/containers based on the Pod and Container specified below. |
+| `cluster_name` | `""`  | Cluster Name to be added to a resource label |
+| `pod_name` | `mongodb` | PThe pod name (without the unique identifier on the end) |
+| `container_name` | `"*"` | The container name of the Mongodb container |
```

Comment:
It might be worth noting that these only matter when source is set to kubernetes. In the observiq cloud ui, these options are hidden when source is file, which is why the descriptions are not more detailed.

I would also move them to the bottom, as most users will not use them.

Reply:
Added `Only relevant if the source is "kubernetes".` to those three parameter descriptions